operations/helm/charts/mimir-distributed/ci/test-enterprise-configmap-values.yaml

```diff
@@ -4,6 +4,33 @@ kubeVersionOverride: "1.20"
 # TODO remove when it is the default
 configStorageType: ConfigMap
 
+global:
+  extraEnvFrom:
+    - secretRef:
+        name: mimir-minio-secret
```

Comment:
Any reason not to use the secret created by the minio subchart chart? 

https://github.com/grafana/mimir/blob/c96a0aed96f0aa3af4bd230f6334707cf8552187/operations/helm/tests/test-enterprise-configmap-values-generated/mimir-distributed/charts/minio/templates/secrets.yaml

Reply:
The standard way to name environment variables is to use capital letters. We can map accessKey in the existing secret to a good name, but it's a bit complicated: https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables. I'll make the change in a commit and we can check if it looks good.

Reply:
I'd rather have this version because I'd like to document this way, since it's a bit simpler than having to do an extra mapping step.